doc/release/RELEASE-NOTES.md

```diff
@@ -19,6 +19,7 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
 
 ### Code/API changes
 * [OSDEV-1006](https://opensupplyhub.atlassian.net/browse/OSDEV-1006) - Create new "api/v1/production-locations" endpoint.
+* [OSDEV-982](https://opensupplyhub.atlassian.net/browse/OSDEV-982) - Search, API. Add OS ID query parameter to v1/production-locations.
```

Comment:
The `api/v1/production-locations/{os_id}` endpoint has also been implemented. Please update the RELEASE-NOTES.md file accordingly.